pkg/kubelet/cm/dra/plugin/dra_plugin_test.go

```diff
@@ -184,6 +185,23 @@ func TestGRPCConnIsReused(t *testing.T) {
 	// We should have only one entry otherwise it means another gRPC connection has been created
 	require.Len(t, reusedConns, 1, "expected length to be 1 but got %d", len(reusedConns))
 	require.Equal(t, 2, reusedConns[conn], "expected counter to be 2 but got %d", reusedConns[conn])
+
+	// Test that health API also reuses the same connection
+	ctx, cancel := context.WithTimeout(tCtx, 5*time.Second)
+	defer cancel()
+
+	stream, err := plugin.NodeWatchResources(ctx)
+	require.NoError(t, err, "Health stream should work")
+	require.NotNil(t, stream)
+
+	// Verify same connection is still being used
+	require.Equal(t, conn, plugin.conn, "Health API should reuse the same connection")
+
+	// Verify stream works
+	resp, err := stream.Recv()
+	require.NoError(t, err)
+	require.NotNil(t, resp)
+	require.Len(t, resp.Devices, 1)
```

Comment:
This test tested also if DRA APIs work as far as I remember. Now that's gone for some reason. Please, try to find a previous PR edition and carefully compare what you had there with this content, thanks!

Reply:
That's right, and I've also put that back.
Please check if the code is correct for your memory.
Thank you.